tests/test_smoke.py

```diff
@@ -591,6 +596,23 @@ def test_file_mounts(generic_cloud: str):
     run_one_test(test)
 
 
+@pytest.mark.lambda_labs
+def test_lambda_file_mounts():
```

Comment:
TODO: merge this with 'test_file_mounts' when multi-node is supported.

Reply:
Added!